### PR TITLE
CI: add community ansible-test images

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Now that we have some more OS-specific images (https://github.com/ansible-community/images/tree/main/ansible-test), let's try to use them :-)

(`tests/utils/shippable/linux-community.sh` is a copy of `tests/utils/shippable/linux.sh` with `"${image}"` replaced by `"quay.io/ansible-community/test-image:${image}"`.)

##### ISSUE TYPE
- Test Pull Request

##### COMPONENT NAME
CI

### CODE REVIEW

**Comment by felixfontein** [comment]
The Debian and ArchLinux images need to be rebuilt before they work. The CentOS Stream 8 error looks really strange.

**Comment by dmsimard** [comment]
I re-ran the jobs: both centos-stream8 and debian-bullseye pass but archlinux fails on a ssh task inside integration tests: https://dev.azure.com/ansible/community.crypto/_build/results?buildId=35973&view=logs&jobId=aeb8cfeb-6801-5365-58ad-9036f81f6479&j=aeb8cfeb-6801-5365-58ad-9036f81f6479&t=8045458f-988f-5dfc-9acf-0fe595069751

File: tests/integration/targets/setup_python_info/vars/main.yml [hunk 1]
```diff
@@ -52,6 +52,12 @@ system_python_version_data:
     '15':
       - '2.7'
       - '3.6'
+  Archlinux:
+    'NA':
+      - '3.10'
+  Debian:
+    '11':
+      - '2.9'
```
Comment (dmsimard) [inline tests/integration/targets/setup_python_info/vars/main.yml:60]:
```suggestion
      - '3.9'
```

**Comment by felixfontein** [comment]
It's actually fails in a setup step by ansible-test itself. Luckily I can reproduce it locally, I guess it's related to `needs/httptester` for the `get_certificate` test.

**Comment by felixfontein** [comment]
https://github.com/ansible-community/images/pull/13 fixes the ArchLinux image.

**Comment by felixfontein** [comment]
If anyone wonders why I added the new images only for ansible-core devel branch tests: I think that's probably good enough coverage, without blowing the CI matrix (and thus costs) much further. If someone disagrees please mention this :)

**Comment by dmsimard** [comment]
@felixfontein I've updated the archlinux image and re-triggered jobs

**Comment by felixfontein** [comment]
@dmsimard thanks! This time it failed because of a wrong package name, let's see how far it gets this time :)

**Comment by felixfontein** [comment]
@dmsimard thanks for your work on this!

**Pull request merged**